### ISSUE TITLE
crashes after update openssl and libssl1.1

### ISSUE THREAD

**Issue by PKonlinegithub** [JohnvandeVrugt/toonapilib4domoticz#19]
Hello, after an automatic update of apt-get on raspberrypi 3B, toonapilib4domoticz crashes in my domoticz. I run python 3.7.3 (toonapilib3.2.3 pip3 install) with beta version 4.11.020 and the latest version of raspberry dabian buster, it is described on the forum that it is caused by openssl and libssl1.1.1.
I think a lot of people suffer from this.
Please help, because I don't know what to do without your nice tool :-)
commandline everything works fine:
python3.7 ./testtoonapilib.py
Starting toonapilib test
The room temperature will be presented on success
Trying to create a show object
Room temperature: 25.0

**Comment by darkrain-nl** [comment]
Did you also open an issue on the Domoticz forum? I ask because it seems this is a Python plugin wide problem and not just this plugin...

**Comment by PKonlinegithub** [comment]
There is already an article about this topic. https://www.domoticz.com/forum/viewtopic.php?f=4&t=28647#p219015
but they only discuss temporary solutions. I did try this but after installing the two plugins my raspberyy got stuck after rebooting. That is why I have put my question to you.

**Comment by darkrain-nl** [comment]
That topic on the forum is not about version 1.1.1 but about 1.1.0j to 1.1.0k. So this seems unrelated to that.

I did try with Raspbian Buster a week or two ago and ran into the same issue you experience.

The solution for the issue wit 1.1.0k was fixed by gizmocuz by linking that version on the build system. My guess would be something similar should be done to fix it for version 1.1.1. That is why I said it is a plugin system wide issue and not only this plugin.

So I hope @JohnvandeVrugt can do something here, but I don't think he can (no offense John).
